src/robusta/integrations/git/git_repo.py

```diff
@@ -12,7 +12,7 @@
 from robusta.core.model.env_vars import GIT_MAX_RETRIES
 
 GIT_DIR_NAME = "robusta-git"
-REPO_LOCAL_BASE_DIR = os.path.join(os.environ.get("REPO_LOCAL_BASE_DIR", "/app"), GIT_DIR_NAME)
+REPO_LOCAL_BASE_DIR = os.path.abspath(os.path.join(os.environ.get("REPO_LOCAL_BASE_DIR", "/app"), GIT_DIR_NAME))
```

Comment:
can you explain?

Reply:
I don't recall what the issue was, but added it to fix an issue with relative paths.